resources/packs/gcp/gcp.lr

```diff
@@ -556,43 +559,234 @@ private gcp.storage.bucket @defaults("id") {
   iamConfiguration dict
 }
 
-// GCP Cloud SQL
-gcp.sql {
-  //
-  instances() []gcp.sql.instance
+// GCP Cloud SQL Resources
+private gcp.project.sqlService {
+  // Project ID
+  projectId string
+  // The list of Cloud SQL instances in the current project
+  instances() []gcp.project.sqlService.instance
 }
 
 // GCP Cloud SQL Instance
-private gcp.sql.instance @defaults("name") {
-  // Instance name
-  name string
+private gcp.project.sqlService.instance @defaults("name") {
+  // Project ID
+  projectId string
+  // All maintenance versions applicable on the instance
```

Comment:
Hi @imilchev! What does "applicable" mean in this case? Newer maintenance versions that could be applied but have not?

Reply:
Honestly, I am not sure. This is all the docs they provided from Google.
```go
// AvailableMaintenanceVersions: List all maintenance versions
// applicable on the instance
AvailableMaintenanceVersions []string `json:"availableMaintenanceVersions,omitempty"`
```

not sure what to make out of that